pom.xml

```diff
@@ -1329,7 +1329,7 @@
                   <version>${java.version}</version>
                 </requireJavaVersion>
                 <requireMavenVersion>
-                  <version>3.3.9</version>
+                  <version>3.8.6</version>
```

Comment:
to unblock this PR, please revert this line.
it is blocked by https://github.com/checkstyle/checkstyle/issues/13970 and it is heavy lifting operation as you can see in related PRs.

we can merge all other changes meanwhile.